internal/scorecard/alpha/config_test.go

```diff
@@ -30,11 +30,12 @@ func TestInvalidConfigPath(t *testing.T) {
 	for _, c := range cases {
 		t.Run(c.configPathValue, func(t *testing.T) {
 			_, err := LoadConfig(c.configPathValue)
-			if err != nil && c.wantError {
-				t.Logf("Wanted error and got error : %v", err)
-				return
-			} else if err != nil && !c.wantError {
-				t.Errorf("Wanted result but got error: %v", err)
+			if err == nil && c.wantError {
+				t.Fatalf("Wanted error but got no error")
+			} else if err != nil {
+				if !c.wantError {
+					t.Fatalf("Wanted result but got error: %v", err)
+				}
 				return
```

Comment:
WDYT about the default one: 

```
   if (err != nil) != c.wantError {
	t.Errorf("TestInvalidConfigPath error = %v, wantErr %v", err, tt.wantErr)
	return
   }
```

Reply:
yeah, not sure,  for some reason that syntax I find hard to read.  I'll think about it however.

Reply:
I am ok if you would prefer this way 👍 all fine .. just to clarifies .. it is  that I found a little weird since usually, we see something as : 

```
func TestAbs(t *testing.T) {
    got := Abs(-1)
    if got != 1 {
        t.Errorf("Abs(-1) = %d; want 1", got)
    }
}
```